packages/config-array/src/config-array.js

```diff
@@ -386,19 +383,12 @@ function shouldIgnorePath(ignores, filePath, relativeFilePath) {
  * Determines if a given file path is matched by a config based on
  * `ignores` only.
  * @param {string} filePath The absolute file path to check.
- * @param {string} basePath The base path for the config.
+ * @param {string} relativeFilePath The relative path of the file to check.
```

Comment:
Is this relative to the base path still? If so, can you add that to the comment?

Reply:
Good point! I looked at `rollup-plugin-node-polyfills`, the polyfills we use in eslint/eslint, and there is no implementation for `toNamespacedPath` in the [`path`](https://github.com/ionic-team/rollup-plugin-node-polyfills/blob/master/polyfills/path.js) module polyfill, so I added a fallback to return the argument as is when `path.toNamespacedPath` is not defined. This is the same as the implementation on Posix, so basically on every non-Windows system ([source](https://github.com/nodejs/node/blob/92a25abca9dd23a723008d88b1ccb63ff40ec9b7/lib/path.js#L1293-L1300)).